implants/lib/eldritch/src/runtime.rs

```diff
@@ -28,7 +28,7 @@ use std::time::Duration;
  */
 #[derive(ProvidesStaticType)]
 pub struct Runtime {
-    stdout_reporting: bool,
+    pub stdout_reporting: bool,
```

Comment:
this doesn't need to be pub, see [with_stdout_reporting](https://github.com/spellshift/realm/blob/main/implants/lib/eldritch/src/runtime.rs#L217)